### ISSUE TITLE
Apply / OK buttons not working when options saved

### ISSUE THREAD

**Issue by mikolajek** [AddonLab/AiOS#118]
AiOS 7.2.9 + FF48 on Windows 10 Anniversary x64 - when any option is changed in AiOS settings both Apply and OK buttons don't work when clicked, so the options can't be saved. At the same time Cancel button works perfectly and closes settings window.

**Comment by JudgmentDay** [comment]
Not sure about Windows 10, but I have Windows 7. For me, while the same effect of the OK and Apply buttons are not automatically closing the window, the changes are applied. You have to click the [x] on the upper right as another way to close the options window.

**Comment by zetavu** [comment]
Not only do those buttons not work but the changes made do not always apply. AiOS 7.29 and FF48 are completely broken, I cannot get sidebar to open. When I downgrade to 7.28 or 7.27 in FF48, sidebar works but not well, history etc open in a new window not in panel, regardless of what I change in settings. I had to downgrade FF to 47.0.1 and AiOS to 7.27 (had issues with 7.28, not sure why) to get back functionality. Those versions won't work with FF 48 so it is an FF 48 issue. Tried in both W10 and W7 on separate machines. Note, sometimes you had to restart FF to get the issue, may have required a reboot. Made it harder to diagnose.
